include/openthread/instance.h

```diff
@@ -193,6 +193,17 @@ otInstance *otInstanceInit(void *aInstanceBuffer, size_t *aInstanceBufferSize);
 otInstance *otInstanceInitSingle(void);
 
 /**
+ * This function indicates whether or not the instance is valid, i.e., it has been initialized and not yet finalized.
+ *
```

Comment:
Would be useful to expand this comment to indicate how the state of being initialized is actually determined.